applications/tari_console_wallet/src/main.rs

```diff
@@ -107,6 +108,11 @@ fn main_inner() -> Result<(), ExitCodes> {
     // get base node/s
     let base_node_config = runtime.block_on(get_base_node_peer_config(&config, &mut wallet))?;
     let base_node = base_node_config.get_base_node_peer()?;
+    let peer_seed_public_keys: Vec<CommsPublicKey> = base_node_config
+        .peer_seeds
+        .iter()
+        .map(|f| f.public_key.clone())
+        .collect();
```

Comment:
These nodes are already available in the `base_node_config`, which is a `PeerConfig` already passed in to `start_wallet`. So you can use this logic to get the public keys using that where you need it 

![image](https://user-images.githubusercontent.com/351403/113850739-10a66f00-979b-11eb-8d8d-8bb078de3606.png)

Reply:
Good observation! Saving one level of functions here then.